app/models/user.rb

```diff
@@ -56,7 +56,7 @@ class User < ActiveRecord::Base
 
 #  Before validation need to fix cell phone numbers
 #  validates_format_of :cell_phone, with: /\A\+?[0-9x]+$\z/, allow_blank: true
-  validates_format_of :screen_name, with: /[^\/]+/
+  validates_format_of :screen_name, with: /\A\+?[a-zA-Z0-9_\-]+$\z/
```

Comment:
I checked it because I wasn't entirely sure, but it breaks the editing of all existing users whose screen name contains some other characters, meaning they cannot update their profile ever.

We counted them a few months back and there were quite a few with dots in their name for example, but I fear I didn't save the full statistics.